### PR TITLE
changed surpath community to surpath hub

### PR DESCRIPTION
**Pull request by bunnyrockz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,7 +67,7 @@ <h2 class="section-title">About</h2>
                 </div>
 
                 <div>
-                    <h2 class="about__subtitle">SurPath Community</h2>
+                    <h2 class="about__subtitle">SurPath Hub</h2>
                     <span class="about__profession">Open Source Projects</span>
                     <p class="about__text">SurPath is an welcoming and inclusive open source community that helps every aspiring student or career shifter in the Philippines. We are here to learn, get along with your passion and know your expertise in the field. Members of
```
Comment (vinzvinci) [inline index.html:72]: Can you add **Hub** next to SurPath

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Can you also add **Hub** in the footer.

Ex. 
`© 2021 SurPath. All right reserved`

File: index.html (hunk 1 above)
Comment (bunnyrockz) [reply index.html:72]: sure

**Review by vinzvinci — APPROVED** [review]
Looks good to me, I'll do the rest.

**Pull request merged**